tests/foreman/api/test_role.py

```diff
@@ -12,7 +12,7 @@
 
 :CaseComponent: UsersRoles
 
-:Assignee: dsynk
+:Assignee: sganar
```

Comment:
Perhaps you should take responsibility for the roles, users, and user groups components in tests/upgrades as well